templates/plugin/remote_execution_ssh.yml.erb

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: <%= @module_enabled %>
 :ssh_identity_key_file: <%= scope.lookupvar('::foreman_proxy::plugin::remote_execution::ssh::ssh_identity_path') %>
+:local_working_dir: <%= @local_working_dir %>
```

Comment:
You need to use scope.lookupvar here, as they're not necessarily in scope on Puppet 4.